src/exponential.jl

```diff
@@ -13,10 +13,40 @@ Reverse function for `exp`.
 function exp_rev(y::Interval, x::Interval)
     y_new = y ∩ (0..∞)
     x_new = x ∩ log(y_new)
+    return y_new, x_new
+end
+
+doc"""
+Reverse function for `exp2`.
+"""
+function exp2_rev(y::Interval, x::Interval)
+    y_new = y ∩ (0.0..∞)
```

Comment:
Is this line actually necessary? Shouldn't `y` already be positive? (It may be necessary, I'm not sure.)

Reply:
A negative domain for y will force x to be an empty interval. So I'd agree its not necessary. I think I just got into a pattern here. I'll remove this.